### PR TITLE
Add configurable NFS mounts for internal read-only access to raw data

### PR DESCRIPTION
**Pull request by manics**
See:
- https://trello.com/c/GKEgLjPb/51-request-nfs-biostudies-and-mount

### CODE REVIEW

File: ansible/idr-ebi-nfs.yml [hunk 1]
```diff
@@ -36,6 +37,5 @@
 
 
   vars:
-    nfs_mountpoint: /nfs/bioimage
     nfs_filesets_dir: /nfs/bioimage/drop/complete
```
Comment (sbesson) [inline ansible/idr-ebi-nfs.yml]: What would you do about this variable? In the absence of `nfs_mountpoint` it still feels pretty specific to the current IDR deployment.

Comment (manics) [reply ansible/idr-ebi-nfs.yml]: It is very specific, but I couldn't immediately see how it fits together with https://github.com/openmicroscopy/management_tools/blob/master/idr/playbooks/idr-links.yml so left it as it is for now

Comment (manics) [reply ansible/idr-ebi-nfs.yml]: For reasons lost to the distant past these seem to be redundant. I've removed them.

**Comment by sbesson** [comment]
Thanks @manics. Deployed on `test51` and the expected shares are now correctly mounted. I can confirm 41c587d has no impact on the deployment either.

**Pull request merged**